packages/convert-exports.js

```diff
@@ -0,0 +1,113 @@
+const glob = require('glob')
+const fs = require('fs')
+
+var getDirectories = function (callback) {
+  glob('./web/src/**/*', callback)
+}
+
+const DIRECTORY = 'hooks'
+const EXPORTS = [
+  'useBooleanOnce',
+  'FEATURE_FLAG_OVERRIDE_KEY',
+  'useRecomputeToggle',
+  'createUseFeatureFlagHook',
+  'useFeatureFlag',
+  'createUseRemoteVarHook',
+  'useRemoteVar',
+  'ButtonState',
+  'ButtonType',
+  'useCurrentStems',
+  'useDownloadTrackButtons',
+  'useImageSize',
+  'useLoadImageWithTimeout',
+  'useInstanceVar',
+  'createUseLocalStorageHook',
+  'createUseTikTokAuthHook',
+  'useTwitterButtonStatus',
+  'useUIAudio',
+  'useSelectTierInfo',
+  'useGetFirstOrTopSupporter',
+  'useRankedSupportingForUser',
+  'createProxySelectorHook',
+  'useProxySelector',
+  'useAccountHasClaimableRewards',
+  'useGatedContentAccess',
+  'useGatedContentAccessMap',
+  'useStreamConditionsEntity',
+  'useLockedContent',
+  'useDownloadableContentAccess',
+  'useLinkUnfurlMetadata',
+  'useThrottledCallback',
+  'useDebouncedCallback',
+  'useAccountAlbums',
+  'useAccountPlaylists',
+  'useFetchedSavedCollections',
+  'useCanSendMessage',
+  'useSetInboxPermissions',
+  'usePlayTrack',
+  'usePauseTrack',
+  'useToggleTrack',
+  'useGeneratePlaylistArtwork',
+  'useUSDCBalance',
+  'useTotalBalanceWithFallback',
+  'usePurchaseContentFormConfiguration',
+  'useChallengeCooldownSchedule',
+  'useAudioMatchingChallengeCooldownSchedule',
+  'useUSDCPurchaseConfig',
+  'usePurchaseContentErrorMessage',
+  'usePayExtraPresets',
+  'getExtraAmount',
+  'isTrackPurchaseable',
+  'PayExtraPreset',
+  'CUSTOM_AMOUNT',
+  'AMOUNT_PRESET',
+  'PURCHASE_METHOD',
+  'PURCHASE_VENDOR',
+  'PAGE',
+  'minimumPayExtraAmountCents',
+  'maximumPayExtraAmountCents',
+  'CENTS_TO_USDC_MULTIPLIER',
+  'DEFAULT_PURCHASE_AMOUNT_CENTS',
+  'PurchaseContentSchema',
+  'useAccessAndRemixSettings',
+  'useInterval',
+  'useCreateUserbankIfNeeded',
+  'usePurchaseMethod',
+  'useCoinflowAdapter',
+  'useIsWaitingForValidation'
+]
+
+getDirectories(async function (err, res) {
```

Comment:
JSDoc explaining this would be nice

Reply:
Sorry these were just temporary scripts, deleted them!